scripts/release/install.js

```diff
@@ -25,6 +27,18 @@ module.exports = {
         }
       }
     )
+    createZip(
```

Comment:
Would you please chain this with createDMG, and make sure that calling the release script returns an exit error if either of them fail?